pipdeptree.py

```diff
@@ -418,6 +466,12 @@ def main():
                             '"raw" output that may be used by external tools. '
                             'This option overrides all other options.'
                         ))
+    parser.add_argument('--output', dest='output_format', required=False,
```

Comment:
`required=False` is redundant here. Also, you may be able to use the `choices` keyword arg to validate the output formats (refer: https://docs.python.org/2.7/library/argparse.html#choices)